### ISSUE TITLE
Replace textAngular with plaintext/md editor

### ISSUE THREAD

**Issue by let4be** [lavab/web#738]

**Comment by piggyslasher** [comment]
@let4be - please discuss with @vLooz and I regarding this. Thanks.

**Comment by vLooz** [comment]
brief summary of a longer discussion yesterday evening:

@pzduniak wants a dropdown button in the text-edit-bar, providing a plaintext/md editor. This editor is for html skilled users, who want to edit their mail text with commands, their are used to. 
@piggyslasher and myself are a bit worried, that this could make the interface too complicated and hard to use for all non-technical users. 

I figured during the discussion that we might have a deep misunderstanding regarding our main long-term target group. 
Vik an I believe, our main target group are privacy-interested, but normal, non-technical, everbody-kind people, who didn't use encryption in the past, because it was too much of a hazzle for them. Our early-adopters are indeed technical, security-savvy people, activists, hacktivists and journalists, although we don't design and develop for them specifically, but want offer "secure email for everyone", like our slogan says. 
From the dev-side I had the impression yesterday, that the dominant opinion there is, that other developers and tech-savvy people are our main target group instead and that we should choose the features we offer by focusing on the needs of this group of users. 

Imo we need some clarification here asap @flixi

**Comment by pzduniak** [comment]
No, I want a dropdown button with three choices: plaintext (no formatting), "standard" (html) and markdown. The HTML editor would be pretty similar to what we have now (but i'd really suggest not dumping DOM into the email body, as the result is garbage half of the time), plaintext would be the same as "no formatting" in Gmail (typed with fixed width font) and markdown would be a split view similar to Ghost's editor.

**Comment by let4be** [comment]
related discussion is here: https://github.com/lavab/web/issues/805

**Comment by piggyslasher** [comment]
https://github.com/bevacqua/woofmark/issues/5#issuecomment-108526162

**Comment by piggyslasher** [comment]
@pzduniak please check out woofmark. its what you want.

**Comment by pzduniak** [comment]
Yea, that's exactly what I want :)

**Comment by piggyslasher** [comment]
+ 1 for woofmark. I think its the cleanest and best I've seen so far.
